src/web/assets/cp/src/css/_cp.scss

```diff
@@ -934,6 +934,13 @@ li.breadcrumb-toggle-wrapper {
 }
 
 #page-title {
+  --page-title-columns: 1fr 34px;
```

Comment:
Added a CSS variable in case plugin authors need to tweak the columns up there. I don't think it's a designated extension point though, so maybe it's not necessary.